add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/java/dev/flutter/multipleflutters/DoubleFlutterActivity.kt

```diff
@@ -0,0 +1,76 @@
+package dev.flutter.multipleflutters
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.FrameLayout
+import android.widget.LinearLayout
+import androidx.fragment.app.FragmentActivity
+import androidx.fragment.app.FragmentManager
+import io.flutter.embedding.android.FlutterFragment
+import io.flutter.embedding.engine.FlutterEngineCache
+
+/**
+ * An activity that displays 2 FlutterFragments vertically.
+ */
+class DoubleFlutterActivity : FragmentActivity(), EngineBindingsDelegate {
+    private val topBindings : EngineBindings by lazy {
+        EngineBindings(this, this, "topMain")
```

Comment:
you can use named arguments to disambiguate

Reply:
done